src/routes/home.js

```diff
@@ -1,95 +1,81 @@
-          style={{ backgroundImage: `url(${printing_home_photo})` }}
-        >
-          <h1 className="text-2xl sm:text-4xl md:text-5xl lg:text-6xl xl:text-7xl 2xl:text-8xl text-white p-10 overflow-hidden">
-            3D Printing Made Local
-          </h1>
-          <Link to="/create" className="bg-custom-purple hover:bg-custom-purple-hover border-black text-white font-bold py-3 px-4 rounded-lg text-2xl sm:text-1xl md:text-2xl lg:text-3xl mb-10">
-            Create An Order
-          </Link>
+      <div className="bg-center bg-cover bg-no-repeat" style={{backgroundImage: `url(${banner_photo})`, height: '100vh'}}>
+        <div style={{backgroundColor: 'rgba(0,0,0,0.5)', top: '0', bottom: '0', left: '0', right: '0', position: 'absolute', zIndex: '1'}}></div>
+        <div className="flex flex-col h-full items-center justify-center gap-4 relative" style={{zIndex: '2'}}>
+          <h2 className="text-white text-8xl font-bold mb-6">3D Printing Made Local</h2>
+          <div>
+            <a className="p-4 py-3 bg-brand-blue rounded text-white mr-3 text-xl font-bold inline-block w-[280px] text-center" href="#iwantprint">I have something to print</a>
+            <a className="p-4 py-3 bg-brand-blue rounded text-white text-xl font-bold inline-block w-[280px] text-center" href="#ihaveprinter">I have a 3D printer</a>
+          </div>
         </div>
       </div>
-      <div className="text-center mt-5">
-        <h1 className="text-4xl font-extrabold">How It Works</h1>
-        {renderSteps()}
-      </div>
-      <div className="flex justify-center mt-10 mb-10">
-        <div className="w-4/5 h-0.5 bg-gradient-to-r from-purple-400 to-purple-600"></div>
-      </div>
-      <div className="text-center mt-5">
-        <h1 className="text-4xl font-extrabold">Own a 3D Printer?</h1>
-      </div>
-      <div className="flex flex-col sm:flex-row mt-5">
-        <div className="flex flex-col sm:flex-row items-center">
-          <div className="order-2 sm:order-1 mt-5 sm:mt-0">
-            <p className="ml-10 text-lg font-medium">
-              Select a job in your local area and start making extra money! It’s
-              as easy as that.
-              <br />
-              <br />
-              Our mission at PrintLink3D is to connect customers with local 3D
-              printing enthusiasts. Interested?
-            </p>
-            <div className="flex justify-center">
-              <Link to="/browse" className="mt-10 bg-custom-purple hover:bg-custom-purple-hover border-black text-white font-bold py-3 px-4 rounded-lg text-2xl sm:text-1xl md:text-2xl lg:text-3xl xl:text-4xl 2xl:text-5xl">
-                Get Started
-              </Link>
+      <div className="container m-auto">
+        <div id="iwantprint" className="flex flex-col items-center justify-center py-[100px]">
+          <h2 className="font-bold text-5xl pb-8">Getting Your Model Printed</h2>
+          <div className="flex flex-row gap-12 w-full py-8">
```

Comment:
For responsiveness it'd be nice to have the steps be vertically placed on small screens

Reply:
I fixed the mobile view for the homepage and its nav bar. Starting to look better now! :)